proto.go

```diff
@@ -25,7 +22,7 @@ func newDialMessage(pi peer.AddrInfo) *pb.Message {
 	}
 
 	return msg
-)
+}
```

Comment:
what happened here?

Reply:
git got confused when merging a file that existed with separate histories from the two repos